Automattic/jetpack
# List publicize connections in Jetpack social

This PR lists the Publicize connections in the admin section of the Social Plugin. 

#### Changes proposed in this Pull Request:

Using the global `$publicize` object, we can fetch the list of existing connections and we can pass it to the front-end through `wp_localize_script`

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:

- Checkout this branch and add some Publicize connections to your site. 
- Go to the `/wp-admin/admin.php?page=jetpack-social` and make sure the publicize connections are listed properly. 
- Add another admin to the website, and create a Publicize connection as the other admin and make the connection to be a global one. Come back to the site as the first admin and make sure you're able to see all the connections again. 
-  Try removing the connections and make sure, they disappear.

### The connections being displayed below the admin hero

<img width="698" alt="Screenshot 2022-04-20 at 4 54 22 PM" src="https://user-images.githubusercontent.com/6594561/164220295-7dcdc85b-52a3-4ee4-a6a5-c21a133c022a.png">
